docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -420,10 +406,9 @@ CI Testing
         1. A dangerous module parameter has been deprecated or removed, and code is present to inform the user that they should not use this specific parameter anymore or that it stopped working intentionally.
         2. Module parameters are only used to pass in data from an accompanying action plugin.
 
-  * All entries in ignores.txt MUST have a justification in a comment in the ignore.txt file for each entry.  For example ``plugins/modules/docker_container.py use-argspec-type-path # uses colon-separated paths, can't use type=path``.
-  * Reviewers can block acceptance of a new collection if they don't agree with the ignores.txt entries.
+  * All entries in ``ignore-*.txt`` files MUST have a justification in a comment in the files for each entry.  For example ``plugins/modules/docker_container.py use-argspec-type-path # uses colon-separated paths, can't use type=path``.
```

Comment:
Why remove the second bullet point?

Reply:
Reviewers can do it anyway: not to put a check in the checkbox, not to put +1 for inclusion, raise objection within the default collection inclusion period - plenty of mechanisms:)
So it's an extra statement imo

Reply:
i'll just remove option 3

Reply:
Reformulated, PTAL

Reply:
I don't see where `Reviewers can block acceptance of a new collection if they don't agree with the ignores.txt entries.` or something equivalent ended up.

Reply:
as i wrote a bit above reviewers can block acceptance in case of any discrepancy with the requirements, so it's imo an excess statement

Reply:
Ah, then I misread. I still don't feel good about removing this though.